### DISCUSSION TITLE
[data importer] Is specifics deprecated or already removed?

### DISCUSSION THREAD

**Discussion by Eppin** [firefly-iii/firefly-iii#5475] [Firefly III] [answered]
Hi 👋 

First of all, thanks for this amazing project! It really helps to get some insight of my financial status.

However, yesterday I've updated and moved from the CSV importer to data importer and after a successfull import I discovered that the descriptions of ING and SNS are not what they're supposed to be. After that I saw some documentation mentioning that the `specifics` and thus `IngDescription` and `SNSDescription` are deprecated (but I guess they're already removed)?

Do you have any tips or tricks to still apply this after (or during) and import? My current work-around would be using the API to fetch the newest transactions and do some magic.

Thanks!

PS: the current [premade configurations](https://github.com/firefly-iii/import-configurations) are also out of date, right?

**Comment by JC5** [comment, marked as answer]
Heya!

Glad to hear you like it!

The specifics have been deprecated for a while now and have been removed in the new data importer. All the old import configurations will still work, but the specifics will be ignored.

**Reply by Eppin** [reply]
Whoops! I must admit that's been a while since I've done a Docker image update. Then my suggested work-around would be the only work-around, right? :-)

**Reply by JC5** [reply]
I'm afraid so! You could also use Nordigen or Spectre, but those come with their own privacy policy and it's not said that they will fix descriptions etc.
